lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -137,9 +137,16 @@ class CatalogController < ApplicationController
     ##
     # Configuration for partials
     config.index.partials.insert(0, :index_breadcrumb)
-    
-    config.show.metadata_partials = [:summary_field, :access_field]
-    
+
+    config.show.metadata_partials = [
+      :summary_field,
+      :access_field,
+      :background_field,
+      :scope_and_arrangement_field,
+      :related_field,
+      :admin_info_field
+    ]
```

Comment:
@mejackreed I noticed that we had to add a key to both `Arclight::Engine.config.catalog_controller_field_accessors` and `Blacklight.config.show.metadata_partial` to get things working.  Would it make sense for use to assign `Blacklight.config.show.metadata_partial` to `Arclight::Engine.config.catalog_controller_field_accessors` here, or is it likely that these will diverge in the near future?